styles/output/ap-physics-2e-pdf.css

```diff
@@ -2916,35 +2912,20 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
   text-align: right;
 }
 
-[data-type=composite-chapter]:not(.os-eoc) > [data-type=composite-page] [data-type=question-solution] > .os-divider {
+.os-eob [data-type=question-solution] > .os-divider {
   margin-right: 8px;
 }
 
-[data-type=composite-chapter]:not(.os-eoc) > [data-type=composite-page] [data-type=question-solution] .os-solution-container {
+.os-eob [data-type=question-solution] > .os-solution-container {
   display: table-cell;
   width: 100%;
 }
 
-[data-type=composite-chapter]:not(.os-eoc) > [data-type=composite-page] [data-type=question-solution] .os-solution-container > [data-type=solution-part] {
-  display: table;
-}
-
-[data-type=composite-chapter]:not(.os-eoc) > [data-type=composite-page] [data-type=question-solution] .os-solution-container > [data-type=solution-part] > .problem-letter {
-  display: table-cell;
-  text-decoration: none;
-  color: #000000;
-}
-
-[data-type=composite-chapter]:not(.os-eoc) > [data-type=composite-page] [data-type=question-solution] .os-solution-container > [data-type=solution-part] > .os-divider {
-  margin-right: 8px;
-}
-
-[data-type=composite-chapter]:not(.os-eoc) > [data-type=composite-page] [data-type=question-solution] .os-solution-container > [data-type=solution-part] .os-solution-container {
-  display: table-cell;
-  width: 100%;
```

Comment:
These styles are moved to another shape, which is used only in Organic Chemistry.